README.md

```diff
@@ -113,6 +72,115 @@ next-page-title: "Create a Layout"
 Omit the "prev-page" info for the first page, and the "next-page" info for the
 last page.
 
+## Syntax highlighting
+
+The flutter.io website uses [prism.js](http://prismjs.com/) for syntax
+highlighting. This section covers how to use syntax highlighting, and
+how to update our syntax highlighter for new languages.
+
+### Supported languages
+
+This website can syntax highlight the following languages:
+
+* shell
+* dart
+* html
+* css
+* javascript
+* java
+* objc
+* swift
+
+### Using syntax highlighting
+
+The easiest way to syntax highlight a block of code is to wrap
+it with triple backticks followed by the language.
+
+Here's an example:
+
+````
```

Comment:
these are intentional. Maybe there's a better way to escape the following triple backticks?